api/v1alpha1/perconaservermysql_types.go

```diff
@@ -962,12 +962,10 @@ func (p *PodSpec) GetTopologySpreadConstraints(ls map[string]string) []corev1.To
 
 // Labels returns a standardized set of labels for the PerconaServerMySQL custom resource.
 func (cr *PerconaServerMySQL) Labels() map[string]string {
-	return map[string]string{
-		naming.LabelName:      "percona-server",
-		naming.LabelInstance:  cr.Name,
-		naming.LabelManagedBy: "percona-server-operator",
-		naming.LabelPartOf:    "percona-server",
-	}
+	l := naming.Labels()
+	l[naming.LabelInstance] = cr.Name
+	l[naming.LabelManagedBy] = "percona-server-operator"
```

Comment:
```suggestion
	l[naming.LabelManagedBy] = "percona-server-mysql-operator"
```

Reply:
This change will cause a lot of changes, including changing a lot of compare files. If we want to follow the recommended labels convention, I would like to do this in another PR as part of this task: https://perconadev.atlassian.net/browse/K8SPS-442